package.json

```diff
@@ -8,7 +8,7 @@
     "npm": ">=3.x"
   },
   "dependencies": {
-    "codemirror-blocks": "bootstrapworld/codemirror-blocks"
+    "codemirror-blocks": "bootstrapworld/codemirror-blocks#b3c36e7a9835810b1324503d787e50860d2c89a5"
```

Comment:
I'm pinning the version of codemirror-blocks that gets used to this commit hash. Just pointing to whatever "master" is makes it impossible to figure out when API incompatibilities might have been introduced.